websites/O/ogladajanime/presence.ts

```diff
@@ -0,0 +1,43 @@
+const presence = new Presence({ clientId: "1137362720254074972" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	anime: HTMLElement = document.querySelector("#anime_name_id"),
+	animeicon = document.querySelector(".img-fluid.lozad");
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		startTimestamp: browsingTimestamp,
+		largeImageKey: "https://i.imgur.com/xHce23t.png",
+	};
+
+	if (document.location.pathname === "/main2")
+		presenceData.details = "Przegląda stronę główną";
+	else if (document.location.pathname.includes("/search/name/"))
+		presenceData.details = "Szuka Anime";
+	else if (document.location.pathname.includes("/chat"))
+		presenceData.details = "Rozmawia na chacie";
+	else if (document.location.pathname.includes("/anime_list/")) {
```

Comment:
Since attributes of `document.location` are used many times in this presence, destructure them to make the code more concise.
```ts
const { pathname, href } = document.location;
if (pathname === "/main2") {
  // ...
  example = [{ label: "...", url: href }]
}
```